ripple/src/main/webapp/auth.js

```diff
@@ -176,14 +168,14 @@ function signOutUser(redirectPage="index.html") {
 Passes blobKey to getBlobKey() */
 function getAcctInfo(uid) {
   console.log("Success: getAcctInfo() recognizes uid: " + uid);
-  var name, email, isBusinessOwner, address, blobKey;
+  var userName, email, isBusinessOwner, address, blobKey;
   db.collection("users")
       .where("uid", "==", uid)
       .get()
       .then(function(querySnapshot) {
```

Comment:
```
      .then((querySnapshot) => {
```

Reply:
Fixed. This function is used for Account Settings page and will be modified in the future / migrated to an accountInfo.js file.